### PR TITLE
Fix use of moved vector

### PR DESCRIPTION
**Pull request by naoyam**

### CODE REVIEW

File: csrc/ir_nodes.cpp [hunk 1]
```diff
@@ -817,44 +816,48 @@ SqueezeOp::SqueezeOp(
...
-      TORCH_INTERNAL_ASSERT(
-          id->extent()->isOneInt(),
-          "Can not squeeze dimension(s) with size != 1.");
-    } else {
-      auto in_id = in_dom[i];
-      auto out_id = out_dom[i - num_removed_broadcasts];
-      TORCH_INTERNAL_ASSERT(
-          in_id->sameAs(out_id), "IterDomain does not match in BroadcastOp");
+  // Validate the squeeze flags when this expr is created with
+  // TensorView. Broadcast with TensorIndex only appears after
```
Comment (zasdfgbnm) [inline csrc/ir_nodes.cpp:821]: I think we will never have squeeze with tensor index. Squeeze is lowered into set op.

**Comment by zasdfgbnm** [comment]
Does this PR fix any failure?

File: csrc/ir_nodes.cpp (hunk 1 above)
Comment (naoyam) [reply csrc/ir_nodes.cpp:821]: That's correct. I'll change it to an assertion

**Comment by zasdfgbnm** [comment]
> > Does this PR fix any failure?
> 
> The use of moved vectors was found with clang-tidy

Oh, I see!

**Pull request merged**